contrib/cscope/packages.el

```diff
@@ -0,0 +1,60 @@
+;;; packages.el --- cscope Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq cscope-packages '(evil-jumper
+                        helm-cscope
+                        xcscope))
```

Comment:
`package.el` take care of installing the dependencies of a package for us so I guess that `xcscope` is a dependency for `helm-cscope`. It would explain why it is installed and not deleted as an orphan even if it has not the `init` function.
@bmag Basically if you have no configuration associated to it then you don't have to list it.